### ISSUE TITLE
Human baseline

### ISSUE THREAD

**Issue by vlievin** [jind11/MedQA#3]
Hi! Is there a known human baseline for this dataset (open and closed book)? Or maybe the required score to pass the exam?

**Comment by jind11** [comment]
Nope, but you can use 60 out f 100 score as a reference score, which is the passing score for the Med Exam.

**Comment by jind11** [comment]
Yeap, 60% of accuracy can be considered as the human passing score. For US dataset, this score is quite hard.

**Comment by miraculixx** [comment]
> Yeap, 60% of accuracy can be considered as the human passing score. For US dataset, this score is quite hard.

Can you elaborate on your reasoning here please? It seems a somewhat flawed heuristic to assume human performance on this particular QA dataset [which to my understanding is generated?] would be approximately equivalent to the outcome of humans being tested in actual exams.

**Comment by ogencoglu** [comment]
Is there any update on human baselines?
